### ISSUE TITLE
Giveth dApp Marketing Campaign Suggestions

### ISSUE THREAD

**Issue by surbhiaroraa** [Giveth/giveth-planning#1531]
Hi @oyealmond 
Can you please review [Giveth dApp Marketing Campaign Suggestions](https://docs.google.com/document/d/1tIfZz21vwbfQxDmTKJt-YKW5CXgnJ0QOkTf7m_o8-z8/edit?usp=sharing). I would love to discuss this during the comms call and take team's input before we start executing these.  

cc: @divine-comedian Just to keep you in the loop.

**Comment by oyealmond** [comment]
Hey @surbhiaroraa! Can you give me edit access to all your docs please?

**Comment by surbhiaroraa** [comment]
Update: Working with Nikola and Ashley to understand about Meet the Makers campaigns with the objective of refining and finalising campaigns ensuring nothing overlaps with what we have already been doing.

**Comment by oyealmond** [comment]
Great @surbhiaroraa! Thanks for the follow up! Im so excited to complement and strengthen MTGM efforts with your ideas!

**Comment by surbhiaroraa** [comment]
@oyealmond @NikolaCreatrix @WhyldWanderer 
Is there any document with all the info of MTM at one place? Would love to start working on it.

**Comment by surbhiaroraa** [comment]
@oyealmond 
I have had elaborate discussions with Ashley, Nikola on these campaigns. I want to know what are the next steps? Will you give the final approval for the campaigns or we take that approval in Comms call? After campaigns are finalised, how does the workflow look like - who writes the copy (I can obviously do the first draft) but would need help with editing, who works on graphics for the campaigns? How to set timelines of launch etc wrt campaign?

**Comment by oyealmond** [comment]
hey dear @surbhiaroraa @WhyldWanderer @NikolaCreatrix 

[Here's the notion](https://www.notion.so/giveth/35eb4a31a42e4f1aaaa0c8033d23e3b0?v=b931c941b25e4d8c9b044bd74f0e1141&pvs=4) to review and comment the campaign ideas. 

We should start brainstorming to complement/tweak each one and then plan the execution. 
Let me know your thoughts on the organization!

**Comment by WhyldWanderer** [comment]
Thanks so much for putting it together @oyealmond! 

I have made a few tweaks, added comments and started brainstorming ideas in there. 
There are a few good ideas in there and it aligns with some of the interview initiatives that are happening now. 

Lets schedule a hack session to go through these ideas, define some next steps and put some of them into the backlog to start working on! I will create a doodle so that we can get together next week. Maybe after the MTM sync?

In the mean time, please add any comments or ideas that you may have in the subcategories of each campaign.
We have decided that the GivingTuesday is a go already so put some extra thought into this one... lets come up with some creative ideas!

**Comment by surbhiaroraa** [comment]
Now that we have discussed the marketing campaigns and started implementing a few, this issue is no longer relevant. I am closing this issue.
